enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/provider/OnboardingProvider.java

```diff
@@ -32,4 +32,21 @@ public interface OnboardingProvider {
 
     void sendOtpCode(String userId, String otpCode, boolean resend) throws OnboardingProviderException;
 
+    /**
+     * Provide consent text.
+     *
+     * @param request consent text request
+     * @return consent text
+     * @throws OnboardingProviderException if there is a problem to fetch consent text
+     */
+    String fetchConsent(ConsentTextRequest request) throws OnboardingProviderException;
```

Comment:
_Context object_ approach makes keeping backward compatibility easier. We may add parameters and don't break compatibility.

Reply:
This is a good idea, however we should stick to one approach within this interface, we use two approaches now (see methods above).

Reply:
Yes, I was about make it consinstent in a separate PR.